app/.env.sample

```diff
@@ -0,0 +1,13 @@
+CONTRACTS=../contracts/artifacts
```

Comment:
The `app/.env.sample` is a great standard for documenting the variables you need to run the project. :D 

Can you create something similar in `deploy/`?

Reply:
Great idea, I actually merged both .env files, now we just need one inside the `app` folder.